FIAT/discontinuous_pc.py

```diff
@@ -0,0 +1,119 @@
+# This file is part of FIAT.
+#
+# FIAT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# FIAT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with FIAT. If not, see <http://www.gnu.org/licenses/>.
+
+from FIAT import finite_element, polynomial_set, dual_set, functional, P0, reference_element
+from FIAT.reference_element import (Point,
+                                    DefaultLine,
+                                    UFCInterval,
+                                    UFCQuadrilateral,
+                                    UFCHexahedron,
+                                    UFCTriangle,
+                                    UFCTetrahedron,
+                                    make_affine_mapping)
+from FIAT.P0 import P0Dual
+import numpy as np
+
+hypercube_simplex_map = {Point(): Point(),
+                         DefaultLine(): DefaultLine(),
+                         UFCInterval(): UFCInterval(),
+                         UFCQuadrilateral(): UFCTriangle(),
+                         UFCHexahedron(): UFCTetrahedron()}
+
+class DPC0Dual(P0Dual):
+    def __init__(self, ref_el):
+        super(DPC0Dual, self).__init__(ref_el)
```

Comment:
Class definitions should be separated by 2 lines. You can check that your code is `flake8`-compliant by running (in the FIAT directory) `flake8 .`